airflow/utils/log/secrets_masker.py

```diff
@@ -47,6 +48,8 @@
 )
 """Names of fields (Connection extra, Variable key name etc.) that are deemed sensitive"""
 
+UNMASKED_SECRETS = {'airflow'}
```

Comment:
I'd call it explictly "SKIP_MASKING_FOR_TESTS"

Reply:
or similar

Reply:
Renamed, any better?